lib/persistence.store.cordovasql.js

```diff
@@ -77,7 +77,7 @@ persistence.store.cordovasql.config = function (persistence, dbname, dbversion,
 
     that.transaction = function (fn) {
       return conn.transaction(function (sqlt) {
-        return fn(persistence.db.websql.transaction(sqlt));
+        return fn(persistence.db.sqliteplugin.transaction(sqlt));
```

Comment:
Probably you want to make this choice depend on the value of `persistence.db.implementation`, if that's set to sqlite, you use sqliteplugin, otherwise websql.